spec/seedData.test.js

```diff
@@ -0,0 +1,29 @@
+const seedDataTest = require('../data/seedDataGenerator');
+
+describe('Seed Data Generator Function Testing', () => {
```

Comment:
I know the tests work for you since I saw you run them, but they don't appear on circleci check. Be sure to add this script in the yml in the last line:
**`- run: npm test`** 
I'm assuming you already have saved this script in your package.json. If not just add the following to your package.json under the script section:
**`"test": "jest"`**

Reply:
@nicola24 Just make this change and it should be good, God willing.